src/Microsoft.DotNet.Build.Tasks.net45/project.json

```diff
@@ -8,7 +8,7 @@
     "NuGet.Versioning": "3.6.0-rc-1987",
     "System.Collections.Immutable": "1.1.37",
     "System.Reflection.Metadata": "1.1.0",
-    "System.Runtime.InteropServices.RuntimeInformation": "4.0.0"
+    "System.Runtime.InteropServices.RuntimeInformation": "4.4.0-beta-24806-04"
```

Comment:
While this may fix the api-set issue it might actually cause other issues. In particular it will not be pulling in the closure of other prerelease packages. Can you please ensure that those other packages don't cause issue.